services/domainlist.js

```diff
@@ -84,7 +82,7 @@ class Domainlist {
     let domain;
 
     // removes protocol and get domain
-    if (url.indexOf('://') > -1) {
+    if (url.indexOf('//') > -1) {
```

Comment:
Maybe using a library like https://www.npmjs.com/package/url-parse will work more reliably?

Reply:
Unfortunately, the `url-parse` and `url` packages don't do well with partial user input. My extensive tests: https://github.com/coralproject/talk/pull/988/files#diff-0b9df2d13c97387e2ccee23f13e6435eR29 have validated that it should always be parsed correctly.